.github/workflows/push.yml

```diff
@@ -0,0 +1,95 @@
+name: Entur/Docker/Push
+
+on:
+  workflow_call:
+    inputs:
+      image_artifact:
+        description: "GitHub artifact with Docker image"
+        type: string
+        default: "repo_name"
+      artifact_registry_project_id:
+        type: string
+        description: "Project id for artifact registry"
+        default: "entur-system-1287"
+      environment:
+        type: string
+        description: "GitHub environment to use"
+        default: "prd"
+      dockerfile:
+        description: "Dockerfile to use for build"
+        type: string
+        default: "Dockerfile"
+    outputs:
+      image:
+        value: ${{ jobs.push.outputs.image }}
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    permissions:
+      contents: read
+      id-token: write
+    outputs:
+      image: ${{ steps.push.outputs.metadata }}
+    timeout-minutes: 10
+    steps:
+      - id: set-env
+        shell: bash
+        run: |
+          echo "GHA_DOCKER_BUILD_IMAGE_ARTIFACT_NAME=${{ inputs.image_artifact }}" >> $GITHUB_ENV
+          echo "GHA_GCR_PROJECT_ID=${{ inputs.artifact_registry_project_id }}" >> $GITHUB_ENV
+          echo "WORKLOAD_IDENTITY_PROVIDER=${{ vars.WORKLOAD_IDENTITY_PROVIDER }}" >> $GITHUB_ENV
```

Comment:
interesting, maybe we should just use the one present on `vars` ?

Reply:
Yes. But then we should update the style guide as well. I agree that it does not make much sense to create env variables from vars